opacclient/opacapp/src/main/res/menu/activity_starred.xml

```diff
@@ -8,6 +8,12 @@
         app:showAsAction="ifRoom"
         android:title="@string/share_starred">
     </item>
+    <item
+        android:id="@+id/action_filter_by_tags"
+        android:icon="@drawable/ic_search_24dp"
+        app:showAsAction="never"
```

Comment:
You could change this to `app:showAsAction="ifRoom"` so that it is shown as an action button. The Icon could be changed to the [filter icon](https://material.io/tools/icons/?search=filter&icon=filter_list&style=baseline).